packages/pluggableWidgets/switch-web/CHANGELOG.md

```diff
@@ -0,0 +1,25 @@
+# Changelog
+All notable changes to this widget will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased]
+### Added
+- Add `Brand Secondary` `Style` design-property option.
+- Add `aria-readonly` for improved accessibility.
+
+### Changed
+- Use `div` instead of `small` for improved semantics.
+- Convert `Style` and `Device style` properties to design-properties.
+- Label property converted to system property and moved to common tab.
+- Editability property converted to system property and moved to common tab.
+
+### Removed
+- Removed `input` element.
+- Removed `info` and `inverse` `Style` property options.
+- Removed label width property.
+
+## 3.0.0 DD-MM-YYY
+
+## Older releases
```

Comment:
Indeed, it _should_ be a single document for all releases. However, this file is relevant to the switch widget, which has gone through many releases prior to introducing this changelog. I decided instead of copy-pasting all the notes into this document a link is more efficient :) 

Although I agree, going forward, for this widget and new widgets, all versions should be in the changelog local to the widget.